tools/cell_census_builder/validate.py

```diff
@@ -66,101 +66,115 @@ def validate_all_soma_objects_exist(soma_path: str, experiment_builders: List[Ex
         |   +-- homo_sapiens: soma.Experiment
         |   +-- mus_musculus: soma.Experiment
     """
-    census = soma.Collection(soma_path, context=SOMA_TileDB_Context())
```

Comment:
all of the below is just indentation changes due to use of open() context manager